lib/srv/db/sqlserver/protocol/protocol_test.go

```diff
@@ -109,3 +109,21 @@ func TestRPCClientRequestPartialLength(t *testing.T) {
 	require.Equal(t, "foo3", p.ProcName)
 	require.NoError(t, err)
 }
+
+func TestRPCClientRequestParamNTEXT(t *testing.T) {
+	// Currently the ReadTypeInfo is not parsing the NTEXT contents correctly,
+	// giving a invalid memory access.
+	//
+	// TODO(gabrielcorado): validate this use case and ensure the parameter is
+	// correctly parsed on the driver.
```

Comment:
Is this meant for this PR?
If not, should we create an issue as well?

Reply:
Created! #29411